content/sensu-go/5.10/guides/scale-event-storage.md

```diff
@@ -0,0 +1,168 @@
+
+## Configuring Sensu {#configuring-sensu-for-postgres}
+
+Assuming your Sensu backend is already licensed, the configuration for routing events to Postgres is relatively straight-forward. To do so, we create a `PostgresConfig` resource which describes the database connection as a Data Source Name:
+
+{{< highlight yml >}}
+type: PostgresConfig
+api_version: store/v1
+metadata:
+  name: postgres01
+spec:
+  dsn: "postgresql://sensu:mypass@10.0.2.15:5432/sensu_events?sslmode=disable"
+  pool_size: 20
+{{< /highlight >}}
+
+{{< highlight json >}}
+{
+  "type": "PostgresConfig",
+  "api_version": "store/v1",
+  "metadata": {
+    "name": "my-postgres"
+  },
+  "spec": {
+    "dsn": "postgresql://sensu:mypass@10.0.2.15:5432/sensu_events",
+    "pool_size": 20
+  }
+}
+{{< /highlight >}}
+
+{{< /language-toggle >}}
+
+
+With this configuration written to disk as `my-postgres.yml` we install it using `sensuctl`:
+
+{{< highlight shell >}}
+sensuctl create -f my-postgres.yml
+{{< /highlight >}}
+
+The Sensu backend is now configured to use Postgres for event storage! 
+
+From the web UI or sensuctl you will observe that event history appears incomplete. When Postgres configuration is provided and the backend successfully connects to the database, etcd event history is not migrated. New events will be written to Postgres as they are processed, with the Postgres datastore ultimately being brought up to date with the current state of your monitored infrastructure.
+
+Aside from event history which is not migrated from etcd, there's no observable difference when using Postgres as the event store, and, as of this writing, neither interface supports displaying the PostgresConfig type.
+
+To verify the change was effective, take a look at the [sensu-backend log][4], where you can see that our connection to Postgres was successful:
+
+{{< highlight shell >}}
+{"component":"store","level":"warning","msg":"trying to enable external event store","time":"2019-10-02T23:31:38Z"}
+{"component":"store","level":"warning","msg":"switched event store to postgres","time":"2019-10-02T23:31:38Z"}
+{{< /highlight >}}
+
+We can also use `psql` to verify that events are being written to the `sensu_events` database:
+
+{{< highlight shell >}}
+postgres=# \c sensu_events
+You are now connected to database "sensu_events" as user "postgres".
+sensu_events=# \dt
+             List of relations
+ Schema |       Name        | Type  | Owner 
+--------+-------------------+-------+-------
+ public | events            | table | sensu
+ public | migration_version | table | sensu
+(2 rows)
+sensu_events=# select sensu_entity from events where sensu_check = 'keepalive';
+ sensu_entity 
+--------------
+ i-414141
+ i-424242
+ i-434343
+(3 rows)
+{{ /highlight }}
+
+The above illustrates connecting to the `sensu_events` database, listing the tables in the database, and requesting a list of all entities reporting keepalives.
```

Comment:
When you say "The above," do you mean "The `psql` command" here?

If so, I suggest:
The `psql` command illustrates connecting to the `sensu_events` database, listing the tables in the database, and requesting a list of all entities reporting keepalives.

Reply:
the shell-highlighted code shows commands being run inside a `psql` session which are meant to illustrate that events are being written. If you think its better to move this around or rephrase, thats fine with me 👍

Reply:
Ah ok, I see! I will make a small suggestion.